.github/PULL_REQUEST_TEMPLATE/release_checklist.md

```diff
@@ -1,6 +1,12 @@
 Here are all the steps for the release process. Create a new PR at the beginning and check them off as you go.
+In the course of the release, you'll create and merge several other PRs. They should all be
+separate from this one. This PR exists to track progress on the release and ensure all the steps
+are carried out.
 When you're done, have someone review and merge the PR along with any fixes to the release process automation
 or documentation.
+The only file changes this PR should contain are changes for the *next* release. If there are none,
+this PR may have no file changes at all. Merge it anyway to make it clear that the release process
+is done and that no updates were made for next time.
```

Comment:
Whoa, you can merge, let alone _make_, a PR that has no content?

Reply:
Hrmm... Feels like a PR for something that may not have any changes may not be a good fit for this checklist.

Perhaps it could be an _issue_ template, instead, with any PRs we make during the release process linking back to it. If there are any release instruction changes, then those can be the final PR that closes the issue out. 

How does that strike you?

Reply:
A great idea; realized in https://github.com/habitat-sh/habitat/pull/5853